### ISSUE TITLE
I have some questions

### ISSUE THREAD

**Issue by Lipapaldl** [xl0418/Phydon#7]
Can you tell me if there is a data leakage problem when I use Phydon's dataset for training and use my own partitioned test set? Is Phydon a biological rule-based model or a parametric training model? @xl0418

**Comment by Lipapaldl** [comment]
Are “doubling_h” and ‘growth_tmp’ in “GTDB_tax_trait_repGenome_in_tree_expanded_archaea.csv” the result of Phydon prediction or the real label?

**Comment by xl0418** [comment]
Hi, thank you for your interest in our work. It is possible to have data leakage if some data in your test set are also in our training data. You can check our training data. We sliced the tree by time and retain one clade as the test data and others as the training data. We do this training for every clade given a slicing time. The scripts for the analysis are stored here https://github.com/xl0418/PhydonAnalysis . The devision of the training and test data sets vary according to where we cut the phylogenetic tree. Pls see our paper for details.

**Comment by xl0418** [comment]
In GTDB_tax_trait_repGenome_in_tree_expanded_archaea.csv, those columns are real labels.
